tests/wpt/mozilla/tests/mozilla/range-on-pi-contextual-fragment-crash.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
This could be upstreamed in tests/wpt/tests/html/syntax/serializing-html-fragments instead.